app/lib/sharezone_plus/page/sharezone_plus_page_bloc.dart

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) 2023 Sharezone UG (haftungsbeschränkt)
+// Licensed under the EUPL-1.2-or-later.
+//
+// You may obtain a copy of the Licence at:
+// https://joinup.ec.europa.eu/software/page/eupl
+//
+// SPDX-License-Identifier: EUPL-1.2
+
+import 'dart:async';
+
+import 'package:bloc_base/bloc_base.dart';
+import 'package:rxdart/subjects.dart';
+import 'package:sharezone/sharezone_plus/page/sharezone_plus_page_view.dart';
+import 'package:sharezone/sharezone_plus/subscription_service/purchase_service.dart';
+import 'package:sharezone/sharezone_plus/subscription_service/revenue_cat_sharezone_plus_service.dart';
+import 'package:sharezone/sharezone_plus/subscription_service/subscription_service.dart';
+
+/// A fallback price if the price cannot be fetched from the backend.
+///
+/// On macOS the price is not fetched from the backend because RevenueCat does
+/// not support macOS.
+const kFallbackPrice = '4,99 €';
```

Comment:
> I think we should probably pass this from remote config (not necessarily in this PR).

Agreed. I created #908 for that.

> Also personally adding k in front feels wrong for me (we can keep it for now though).

Just added it because sometimes the Flutter team uses it. However, I don't have a strong option about this and it's also against Dart Effective guidelines. So let's remove it 👍 

https://dart.dev/effective-dart/style#dont-use-prefix-letters